src/components/IstioWizards/WizardHelp.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { KialiIcon } from '../../config/KialiIcon';
+import { style } from 'typestyle';
+
+const infoStyle = style({
+  margin: '0px 0px 2px 10px',
+  verticalAlign: '-5px !important'
+});
+
+const importantTooltip = style({
+  fontWeight: 700
+});
+
+export const wizardTooltip = (tooltipContent: React.ReactFragment) => {
+  return (
+    <Tooltip position={TooltipPosition.right} content={<div style={{ textAlign: 'left' }}>{tooltipContent}</div>}>
+      <KialiIcon.Info className={infoStyle} />
+    </Tooltip>
+  );
+};
+
+export const CONNECTION_POOL_TOOLTIP = (
+  <>
+    <div style={{ marginBottom: 5 }}>Connection pool settings for an upstream host.</div>
+    <div style={{ marginBottom: 5 }}>The settings apply to each individual host in the upstream service.</div>
+    <div>
+      Connection pool settings can be applied at the <span className={importantTooltip}>TCP</span> level as well as at{' '}
+      <span className={importantTooltip}>HTTP</span> level.
+    </div>
+  </>
+);
+
+export const GATEWAY_TOOLTIP = (
+  <>
+    <div style={{ marginBottom: 5 }}>The names of gateways that should apply these routes.</div>
+    <div style={{ marginBottom: 5 }}>
+      If a list of gateway names is provided, the route rules will apply only to the gateways.
+    </div>
+    <div>
+      To apply rules to both gateways and sidecars, specify the reserverd word{' '}
+      <span className={importantTooltip}>mesh</span>.
+    </div>
+  </>
+);
+
+export const HTTP_ABORT_TOOLTIP = (
+  <>
+    Abort Http request attempts and return error codes back to <span className={importantTooltip}>DOWNSTREAM</span>{' '}
```

Comment:
I would change "Http" for "HTTP", it is only an aesthetic change so that it is consistent with the rest of the times it appears.